### PR TITLE
cleanup(driver,libscap): revert back breaking change in #948.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area driver-kmod
/area driver-bpf
/area driver-modern-bpf
/area libscap

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

Add a way to skip PPM_SC_NA_X codes while populating syscall_info_table.
This reverts #948 too. Sorry for the trip :)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
cleanup(driver,libscap): revert #948 and allow to skip unused/wrong PPM_SC codes in future if needed.
```

### CODE REVIEW

File: driver/ppm_events_public.h [hunk 1]
```diff
@@ -1608,19 +1608,20 @@ enum extra_event_prog_code
...
-	PPM_SC_X(GET_MEMPOLICY, 386) \
-	PPM_SC_X(SYNC_FILE_RANGE, 387) \
-	PPM_SC_X(PROCESS_MADVISE, 388) \
-	PPM_SC_X(MEMBARRIER, 389) \
-	PPM_SC_X(IOPL, 390) \
-	PPM_SC_X(CLOSE_RANGE, 391) \
-	PPM_SC_X(FANOTIFY_MARK, 392) \
-	PPM_SC_X(RECV, 393) \
-	PPM_SC_X(SEND, 394)
+	PPM_SC_X(NA_1, 382) \
```
Comment (FedeDP) [inline driver/ppm_events_public.h:1611]: Added back old unused PPM_SC_PIDFD_GETFD calling it `NA_1`.
PPM_SC codes starting with `NA_` will be skipped while popilating syscall info table.

File: driver/ppm_events_public.h [hunk 2]
```diff
@@ -1608,22 +1608,23 @@ enum extra_event_prog_code
...
-	PPM_SC_X(GET_MEMPOLICY, 386) \
-	PPM_SC_X(SYNC_FILE_RANGE, 387) \
-	PPM_SC_X(PROCESS_MADVISE, 388) \
-	PPM_SC_X(MEMBARRIER, 389) \
-	PPM_SC_X(IOPL, 390) \
-	PPM_SC_X(CLOSE_RANGE, 391) \
-	PPM_SC_X(FANOTIFY_MARK, 392) \
-	PPM_SC_X(RECV, 393) \
-	PPM_SC_X(SEND, 394)
+	/*PPM_SC_X(NA_1, 382)*/ \
```
Comment (FedeDP) [inline driver/ppm_events_public.h:1611]: Moreover, we are now able to skip PPM_SC if needed, without worrying.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**